pywren/jobrunner.py

```diff
@@ -41,9 +43,21 @@ def b64str_to_bytes(str_data):
 output_bucket = jobrunner_config['output_bucket']
 output_key = jobrunner_config['output_key']
 
+## Jobrunner stats are fieldname float
+jobrunner_stats_filename = jobrunner_config['stats_filename']
+# open the stats filename
+stats_fid = open(jobrunner_stats_filename, 'w')
+
+def write_stat(stat, val):
+    stats_fid.write("{} {:f}\n".format(stat, val))
```

Comment:
can we also `flush` the file handle here ? it'll make sure full lines are written to the file system asap